db/upgrade.php

```diff
@@ -99,5 +99,18 @@ function xmldb_cms_upgrade($oldversion) {
         upgrade_mod_savepoint(true, 2023072400, 'cms');
     }
 
+    if ($oldversion < 2023072500) {
+        // Define field datasources.
+        $table = new xmldb_table('cms_types');
+        $field = new xmldb_field('isvisible', XMLDB_TYPE_INTEGER, 1, null, true, null, 0, 'customdata');
+
+        // Conditionally launch add field.
+        if (!$dbman->field_exists($table, $field)) {
+            $dbman->add_field($table, $field);
+        }
```

Comment:
Should probably add a step here that pre-populates existing cms_types with the value which should resolve the issue noted above

Reply:
Done